decidim-core/lib/decidim/core.rb

```diff
@@ -510,6 +510,11 @@ def self.reset_all_column_information
     Rails.env.exclude?("development")
   end
 
+  # List of static pages' slugs that can include content blocks
+  config_accessor :page_blocks do
```

Comment:
I would be great if we could add information about this in the ```decidim/decidim-generators/lib/decidim/generators/app_templates/initializer.rb```

Reply:
Applied!